odoo/addons/test_lint/tests/eslintrc

```diff
@@ -33,7 +33,11 @@
             {
                 "selector": "CallExpression[callee.name='_t'] > :first-child[value=/(.*%s){2,}/]",
                 "message": "Usage of _t function with multiple unnamed placeholders"
-            }
+            },
+            {
+                "selector": "CallExpression[callee.object.name='notificationService'][callee.property.name='add'] > Literal",
```

Comment:
You only want to check the first argument:
```suggestion
                "selector": "CallExpression[callee.object.name='notificationService'][callee.property.name='add'] > Literal:first-child",
```
Otherwise, it would also catch this:
```js
notificationService.add(_t("ok"), "this is caught although it shouldn't");
```